### PR TITLE
Add Jellyfish links: INFODEV-2519

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-40a9257e64/how-to/)

**Comment by cyrichardson** [comment]
@RackerKate On it.

File: content/rackspace-email/rackspace-email-and-hosted-exchange-settings.md [hunk 1]
```diff
@@ -5,20 +5,23 @@ title: Rackspace Email and Hosted Exchange settings
...
 ---
 
-To set up your email hosted by Rackspace on either your desktop email
-client (like Outlook or Mac Mail) or on your mobile device, you can
-use our Email Help Tool or enter the settings manually.
+This article helps you set up [Rackspace
+Email](https://www.rackspace.com/email-hosting) or [Hosted
+Exchange](https://www.rackspace.com/email-hosting/hosted-exchange) email on a
+desktop client (such as Microsoft Outlook or Mac Mail), and on your mobile
+device. You can either use our Email Help Tool, or enter the settings manually.
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-email-and-hosted-exchange-settings.md]: device. You can either use our Email Help Tool or enter the settings manually.

File: content/rackspace-email/rackspace-email-and-hosted-exchange-settings.md [hunk 2]
```diff
@@ -5,20 +5,23 @@ title: Rackspace Email and Hosted Exchange settings
...
 product_url: rackspace-email
 ---
 
-To set up your email hosted by Rackspace on either your desktop email
-client (like Outlook or Mac Mail) or on your mobile device, you can
-use our Email Help Tool or enter the settings manually.
+This article helps you set up [Rackspace
+Email](https://www.rackspace.com/email-hosting) or [Hosted
+Exchange](https://www.rackspace.com/email-hosting/hosted-exchange) email on a
+desktop client (such as Microsoft Outlook or Mac Mail), and on your mobile
```
Comment (cyrichardson) [inline content/rackspace-email/rackspace-email-and-hosted-exchange-settings.md]: desktop client (such as Microsoft&reg; Outlook&reg; or Mac Mail), and on your mobile

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a5f9ca547d/how-to/).

**Pull request merged**